test/fixtures/test-runner/output/spec_reporter_cli.snapshot

```diff
@@ -151,7 +149,8 @@
  <anonymous> (*ms) # SKIP
  test with a name and options provided (*ms) # SKIP
  functionAndOptions (*ms) # SKIP
- escaped description \ # \#\ \n \t \f \v \b \r (*ms)
+ escaped description \ # \#\ 
```

Comment:
It looks like some information is being lost?

Reply:
this it now the same as running without `--test`